packages/validator-ajv8/test/harness/compileTestSchema.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * In order to keep things in sync, it may be necessary to run this after making changes in the schemaParser world in
+ * `@rjsf/utils` OR if an AJV update is installed. To run this, simply do the following, starting in the root directory
+ * of the `@rjsf/validator-ajv8` directory:
+ *
+ * - cd test/harness
+ * - node compileTestSchema.js
+ *
+ * Then add the two updated `superSchema.js` and `superSchemaOptions.js` files to your PR
+ */
+
+const compileSchemaValidators = require('../../dist').compileSchemaValidators;
+const superSchema = require('./superSchema.json');
+
+// NOTE these are the same as the CUSTOM_OPTIONS in `testData.ts`, keep them in sync
```

Comment:
Why can't we just import this from `testData`?

Reply:
The TS file is using imports for types which don't work in a CJS world that this file is running in.

Reply:
Gotcha, so this is easier than using ts-node or something like that. That's fine.